scripts/gtk-bundle-from-msys2.sh

```diff
@@ -64,17 +68,25 @@ handle_command_line_options() {
 		"-3")
 			gtkv="3"
 			;;
+		"-4")
+			gtkv="4"
```

Comment:
I find the mentions of gtk4 weird as it suggest we have any support for it which is not true.

Reply:
Having the ability to choose a GTK4 build is probably useful for when "somebody" (:tm: @eht16) starts to add GTK4 support.   For once Windows is leading not trailing :grin:

Reply:
Renaming GTK2 to GTK4 was done to keep the structure in the script, I assume it's not that unlikely that GTK4 will come at some point.

Considering the target audience of this script, I don't see a big source of confusion here. The script is used only by very few people after all (you and me included).